app/src/main/java/com/isc/hermes/model/Utils/IncidentsUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.isc.hermes.model.Utils;
+
+import org.bson.types.ObjectId;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+/**
+ * Utility class for handling incidents-related operations.
+ */
+public class IncidentsUtils {
+    /**
+     * Generates a new ObjectId as a hexadecimal string.
+     *
+     * @return The generated ObjectId as a hexadecimal string.
+     */
+    public static String generateObjectId() {
```

Comment:
Try to avoid static methods. As it is a util, you could make a singleton instance in order to avoid instancing the class so many times and not using static methods.

Reply:
This observation has been resolved.